Xamarin.Forms.Platform.Android/AndroidTicker.cs

```diff
@@ -40,12 +42,24 @@ internal void CheckPowerSaveModeStatus()
 			var powerSaveOn = powerManager.IsPowerSaveMode;
 
 			// If power saver is active, then animations will not run
-			_systemEnabled = !powerSaveOn;
+			_energySaveModeDisabled = !powerSaveOn;
 			
 			// Notify the ticker that this value has changed, so it can manage animations in progress
 			OnSystemEnabledChanged();
 		}
 
+		static bool IsAnimatorEnabled()
+		{
+			var resolver = global::Android.App.Application.Context?.ContentResolver;
+			if (resolver == null)
```

Comment:
Should we fallback to users having animations? IMO this should be the default is nulls appear

Reply:
@bentmar Imho the fallback should be that animations are disabled.

Reason: If the animations are enabled if nulls appear, the app is potentially broken because of the freezing animation tasks.

Reply:
@bentmar Can this conversation be resolved?

Reply:
yes